README.md

```diff
@@ -122,10 +116,16 @@ You may further find it convenient to directly use the search function on
 
 ## hipfc wrapper compiler and Makefile.hipfort
 
-Aside from Fortran interfaces to the HIP and ROCm libraries, hipfort ships the `hipfc` wrapper compiler
-and a `Makefile.hipfort` that can be included into a project's build system. hipfc located in the `bin/` subdirectory and
-Makefile.hipfort in share/hipfort of the repository. While both can be configured via a number of environment variables,`
-hipfc` also understands a greater number of command line options that you can print to screen via `hipfc -h`.
+The hipfc wrapper compiler is deprecated and may be removed in a future release. Users are
```

Comment:
```suggestion
The hipfc wrapper compiler is deprecated and might be removed in a future release. Users are
```

Reply:
We should probably use stronger language. I'll change this to "will".